aws-distro-opentelemetry-node-autoinstrumentation/src/patches/instrumentation-patch.ts

```diff
@@ -57,6 +61,10 @@ export function applyInstrumentationPatches(instrumentations: Instrumentation[])
       if (services) {
         services.set('S3', new S3ServiceExtension());
         services.set('Kinesis', new KinesisServiceExtension());
+        services.set('Lambda', new LambdaServiceExtension());
+        services.set('SNS', new SNSServiceExtension());
+        services.set('SecretsManager', new SecretsManagerServiceExtension());
+        services.set('SFN', new StepFunctionsServiceExtension());
```

Comment:
Please remove SNS and Lambda from this list.

Follow the SQS example instead:
`patchSqsServiceExtension(services.get('SQS'));`